map/src/main/res/layout/fragment_map.xml

```diff
@@ -55,4 +61,13 @@
         app:layout_constraintDimensionRatio="1:1"
         app:layout_constraintEnd_toEndOf="parent" />
 
+    <FrameLayout
+        android:id="@+id/objectDetailsContainer"
+        android:layout_width="0dp"
+        android:layout_height="135dp"
```

Comment:
layout height here should be `wrap_content`. why is this hardcoded?

Reply:
I had to do this so that the audio player is not overlapped.

Reply:
use a `dimen` then.

Reply:
now using dimen.